lib/common.js

```diff
@@ -17,6 +17,23 @@ var Common = function () {
 
 		resolveEnvironmentVariable: function (env) {
 			return process.env[env];
+		},
+
+		toBoolean: function (value) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'toBoolean' has a complexity of 8.](https://www.codacy.com/app/connor-bray/atom-maven/file/2738518164/issues/source?bid=3352650&fileBranchId=3353509#l22)